src/sparseml/pytorch/torchvision/utils.py

```diff
@@ -210,6 +213,7 @@ def accuracy(output, target, topk=(1,)):
         for k in topk:
             correct_k = correct[:k].flatten().sum(dtype=torch.float32)
             res.append(correct_k * (100.0 / batch_size))
+            res.append(correct_k * 100.0)
```

Comment:
@abhinavnmagic Isn't correct_k already the number of correct predictions? I'm confused as to why we need to the multiplication here.

Reply:
I initially didn't multiply by 100 here; and adjusted this in the global_avg function. However, that results in the scaling of losses by 100 (during training run), which is something we don't want. Due to this reason, I moved the scaling to the implementation here. I do agree that this results in this multiplication here, but I don't have any other alternative.

Reply:
ok at least having the scaling here is less confusing than in the global_avg.